### ISSUE TITLE
oc : how to set dockerfile path during new-app/new-build

### ISSUE THREAD

**Issue by robdaems** [openshift/origin#18610]
I've a repo where my Dockerfile is called and located in /docker/Dockerfile.x86_64 (different architectures to build).
The rest of my source is located in /
I know it's possible to set the Dockfile path in the yaml, but I'd like to do it directly when creating the app/build. Now I've to make dummy app/build and adapt the yaml.

Has anyone an idea how to give the dockerfile path when using oc new-app/new-build).

##### Version
oc v3.7.0+7ed6862

**Comment by php-coder** [comment]
@openshift/sig-developer-experience

**Comment by bparees** [comment]
There's no way to do so currently.  new-app/new-build have a limited ability to influence the shape of the buildconfig they create.  Beyond the basic capabilities, the expectation is that you will either:

1) craft your own buildconfig from scratch
2) tweak the buildconfig it produces.

to aid with (2) you can use "oc new-build -o yaml > somefile.yaml", then edit the yaml before doing an "oc create -f somefile.yaml".

Basically we don't want to add a feature flag to new-build for every single api field in the buildconfig, it would become quite unwieldy
